lib/rex/proto/mssql/client_mixin.rb

```diff
@@ -229,6 +230,10 @@ def mssql_parse_tds_reply(data, info)
 
       else
         col[:id] = :unknown
+
+        # See https://learn.microsoft.com/en-us/openspecs/windows_protocols/ms-tds/ce3183a6-9d89-47e8-a02f-de5a1a1303de for details about column types
+        info[:errors] << "Unsupported column type: #{col[:type]}. "
```

Comment:
Should output `col[:type]` as hex value, because that's what Microsoft uses in their reference doc

Reply:
free tds uses integers, which is where I think the original values came from:

https://www.freetds.org/tds.html

Reply:
(I don't mind hex or decimal output here)

Reply:
Makes sense, then let's disregard this suggestion